spec/system/candidate_interface/apply_again/carry_over_unsuccessful_application_after_apply_2_deadline_spec.rb

```diff
@@ -49,19 +49,16 @@ def and_i_visit_my_application_complete_page
     visit candidate_interface_application_complete_path
   end
 
-  def and_i_click_on_apply_again
-    expect(page).to have_content "Courses for the #{RecruitmentCycle.cycle_name(RecruitmentCycle.next_year)} academic year are now closed"
-    click_link 'apply again'
+  def and_i_click_go_to_my_application_form
+    click_link 'Go to your application form'
   end
 
-  def and_i_click_on_start_now
-    expect(page).to have_content "You can submit your application from #{CycleTimetable.apply_opens.to_s(:govuk_date)}."
-    expect(page).to have_content 'Your courses have been removed. You can add them again later.'
-    click_button 'Apply again'
+  def then_i_see_the_carry_over_inset_text
+    expect(page).to have_content 'You can apply for courses starting in the 2021 to 2022 academic year instead.'
```

Comment:
Isn't this the same as on the interstitial — worth checking it's definitely appearing in the inset?

Reply:
Interstitial text is slightly different ...
```
Continue your application to apply for courses starting in the <%= next_academic_cycle %> academic year instead.
```

When we test the interstitial we check the `current_path`

```
  def then_i_am_redirected_to_the_carry_over_interstitial
    expect(page).to have_current_path candidate_interface_start_carry_over_path
  end
```

Reply:
OK!